oath.c

```diff
@@ -216,3 +216,38 @@ oauth_calc(
 
 	return truncatedHash;
 }
+/** C function to do hotp computation */
+uint32_t
+hotp_calc(
```

Comment:
Is this the same function as `totp_calc()`, but using counter instead of now?

would it make sense to add a flag to the `totp` program that allows a user-specified "now", which could be either a clock or the counter?